### ISSUE TITLE
Cannot call method 'toFixed' of undefined 

### ISSUE THREAD

**Issue by juliangarnier** [thomasxiii/photon#3]
I came across something strange, if I apply some rotation to a face element with both rotateX and rotateY set to 90deg, I get this error :

Uncaught TypeError: Cannot call method 'toFixed' of undefined

But if I change one of the rotation values to something else (like 89deg) everything works correctly.

Exemple here : http://jsfiddle.net/JulianG/X7FZA/

At the end of the css, if you replace one of the 90deg values of ".object-3x10 .lt", the problem disappear.

**Comment by piksel** [comment]
I have the same problem but only in Firefox. Webkit-based browsers are fine, but firefox throws `[00:33:23.433] (new TypeError("self[p] is undefined", "photon.js", 712))`
Using 89.999deg works as a temporary fix.

**Comment by juliangarnier** [comment]
@piksel I found a quick fix, can you give it a try?

**Comment by piksel** [comment]
@juliangarnier, it clears the error message, but still no lightning unless I change to 89.99.

using firefox v22.0 (32-bit), on windows 7 64-bit.

**Comment by juliangarnier** [comment]
Weird. Can you provide a jsfiddle ?

Does [this example](http://jsfiddle.net/JulianG/yS8sF/) works on your FF?

**Comment by piksel** [comment]
No, but you only used -webkit-prefixes, so I made a fork: http://jsfiddle.net/NGjjM/

[Heres chrome (left), and firefox 90deg (right top), firefox 89deg (right bottom)](http://i.imgur.com/VALllLA.png)
